articles/java/spring-framework/configure-spring-boot-starter-java-app-with-azure-active-directory-b2c-oidc.md

```diff
@@ -197,18 +188,19 @@ Now that you've created the Azure AD B2C instance and some user flows, you'll co
     azure:
       activedirectory:
         b2c:
-          tenant: ejb0518domain
-          client-id: 11111111-1111-1111-1111-1111111111111111
-          client-secret: '<yourAppKey>'
-          reply-url: http://localhost:8080/home
-          logout-success-url: http://localhost:8080/home
+          base-uri: https://<your-tenant-initial-domain-name>.b2clogin.com/<your-tenant-initial-domain-name>.onmicrosoft.com/
+          client-id: <your-application-id> 
+          client-secret: '<secretValue>'
+          login-flow: sign-up-or-sign-in
+          logout-success-url: <your-logout-success-url> 
           user-flows:
-            sign-up-or-sign-in: B2C_1_signupsignin1
-            profile-edit: B2C_1_profileediting1
-            password-reset: B2C_1_passwordreset1
+            sign-up-or-sign-in: <your-sign-up-or-sign-in-user-flow-name> 
+            profile-edit: <your-profile-edit-user-flow-name> 
+            password-reset: <your-password-reset-user-flow-name> 
+          user-name-attribute-name: <your-user-name-attribute-name> 
     ```
 
-    Notice that the `client-secret` value is enclosed in single quotes. This is necessary because the value of `<yourAppKey>` will almost certainly contain some characters that require being inside single quotes when present in YAML.
+    Notice that the `client-secret` value is enclosed in single quotes. This is necessary because the value of `<secretValue>` will almost certainly contain some characters that require being inside single quotes when present in YAML.
```

Comment:
```suggestion
    Notice that the `client-secret` value is enclosed in single quotes. This is necessary because the value of `<secret-value>` will almost certainly contain some characters that require being inside single quotes when present in YAML.
```